### PR TITLE
Machines start and stop

### PR DESCRIPTION
**Pull request by Vito-Lu**

### CODE REVIEW

File: src/test/java/core/basesyntax/MainAppTest.java [hunk 1]
```diff
@@ -1,5 +1,4 @@
 package core.basesyntax;
 
 public class MainAppTest {
```
Comment (Tarabarnik) [inline src/test/java/core/basesyntax/MainAppTest.java:3]: what is this empty class for?

File: src/main/java/core/basesyntax/Bulldozer.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Start working");
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Bulldozer.java:6]: Lets make the overriden methods to write different text

File: src/test/java/core/basesyntax/MainAppTest.java (hunk 1 above)
Comment (Vito-Lu) [reply src/test/java/core/basesyntax/MainAppTest.java]: fixed

File: src/main/java/core/basesyntax/Bulldozer.java (hunk 2 above)
Comment (Vito-Lu) [reply src/main/java/core/basesyntax/Bulldozer.java]: fixed

**Pull request closed without merging**